UefiPayloadPkg/UefiPayloadPkgIa32X64.dsc

```diff
@@ -94,6 +94,7 @@
   DEFINE LOAD_OPTION_ROMS               = TRUE
   DEFINE SECURE_BOOT_DEFAULT_ENABLE     = TRUE
   DEFINE DASHARO_SYSTEM_FEATURES_ENABLE = FALSE
+  DEFINE SETUP_PASSWORD_ENABLE          = FALSE

```

Comment:
Changes in this file are no longer needed. We do symlink in the PR with Secure Boot

Reply:
Ack

Reply:
Reverted the changes to this file. Of course now it won't work until the secure boot patches get merged.